<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-01-04T14:33:26 PST" name="Default suite" finished-at="2021-01-04T14:33:50 PST" duration-ms="23938">
    <groups>
    </groups>
    <test started-at="2021-01-04T14:33:26 PST" name="Default test" finished-at="2021-01-04T14:33:50 PST" duration-ms="23938">
      <class name="classwork.AssertsDemo">
        <test-method signature="launchBrowser()[pri:0, instance:classwork.AssertsDemo@6e509ffa]" started-at="2021-01-04T14:33:26 PST" name="launchBrowser" finished-at="2021-01-04T14:33:35 PST" duration-ms="8595" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title mismatch expected [MyStore] but found [My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title mismatch expected [MyStore] but found [My Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at classwork.AssertsDemo.launchBrowser(AssertsDemo.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method signature="search()[pri:1, instance:classwork.AssertsDemo@6e509ffa]" started-at="2021-01-04T14:33:35 PST" name="search" finished-at="2021-01-04T14:33:45 PST" duration-ms="9603" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- search -->
        <test-method signature="create()[pri:2, instance:classwork.AssertsDemo@6e509ffa]" started-at="2021-01-04T14:33:45 PST" name="create" finished-at="2021-01-04T14:33:45 PST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- create -->
        <test-method signature="close()[pri:3, instance:classwork.AssertsDemo@6e509ffa]" started-at="2021-01-04T14:33:45 PST" name="close" finished-at="2021-01-04T14:33:50 PST" duration-ms="5698" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- classwork.AssertsDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
